backend/requirements-dev.lock

```diff
@@ -7,6 +7,7 @@
 #   all-features: false
 #   with-sources: false
 #   generate-hashes: false
+#   universal: false
```

Comment:
Pleas run `rye sync` again, or just do not change this file.

Reply:
Changed it back manually